Source/ui_android/java/com/ikimuhendis/ldrawer/ActionBarDrawerToggle.java

```diff
@@ -0,0 +1,182 @@
+package com.ikimuhendis.ldrawer;
+
+
+import android.content.res.Configuration;
+import android.graphics.drawable.Drawable;
+import android.os.Build;
+import android.support.v4.view.GravityCompat;
+import android.support.v4.widget.DrawerLayout;
+import android.support.v7.app.ActionBar;
+import android.support.v7.app.ActionBarActivity;
+import android.util.Log;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+
+import com.virtualapplications.play.R;
```

Comment:
no, as libraries are, you'd import from them and not to them.
however the particular line you reference gives us access to Resources from the main app, in this case few lines lower 'R.drawable.ic_drawer'
but even then you mention the emulator, I’m assuming you mean the main ui of the app and not the emulator activity?